lighthouse-cli/test/smokehouse/test-definitions/lantern/lantern-expectations.js

```diff
@@ -98,4 +98,31 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/ric-shim.html?short',
+      finalUrl: 'http://localhost:10200/ric-shim.html?short',
+      audits: {
+        'total-blocking-time': {
+          // With the requestIdleCallback shim in place 1ms tasks should not block at all and should max add up to
+          // 12.5 ms each, which would result in 50ms under a 4x simulated throttling multiplier and therefore in 0 tbt
+          numericValue: '<=0',
```

Comment:
I think we can give it a little bit of slack to avoid flakiness, maybe `<=100`? going from 5500 -> 100 is still great :)

Reply:
makes sense 👍 done